grails-app/assets/javascripts/customer/customerEdit.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function () {
+  let inputsReference = $("input");
+
+  $(".js-editCustomer").on("click", function (e) {
+    e.preventDefault();
+    $(this).addClass("hiddenBtn");
```

Comment:
não funcionaria assim?
```suggestion
    $(this).hide();
```